.github/workflows/codecov.yml

```
@@ -15,11 +15,11 @@ jobs:
       - name: Install dependencies
         run: go mod tidy
       - name: Run tests with coverage
-        run: go test -timeout 1h -coverprofile="coverage.out" $(go list ./... | grep -v /mocks | grep -v /facades | grep -v /contracts | grep -v /testing/mock)
+        run: go test -timeout 1h -coverprofile="coverage.out" $(go list ./... | grep -v /mocks | grep -v /facades | grep -v /contracts | grep -v /testing/mock | grep -v /database)
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:50: Quote this to prevent word splitting

(shellcheck)